shared/trieutil/sparse_merkle_test.go

```diff
@@ -150,7 +150,7 @@ func TestMerkleTrie_VerifyMerkleProof_TrieUpdated(t *testing.T) {
 		{3},
 		{4},
 	}
-	m, err := GenerateTrieFromItems(items, int(params.BeaconConfig().DepositContractTreeDepth)+1)
```

Comment:
Why change this?

Reply:
probably tie back to @terencechain comment.
As I wasn't passing the depth of the trie as an input of `VerifyMerkleBranch`, the verification always assume a depth of `int(params.BeaconConfig().DepositContractTreeDepth)`.

This test was creating a Trie of depth +1, so I updated it.

Will update the implementation to pass depth as an input per @terencechain comment above.